### PR TITLE
fix(slider): align editable slider when no label provided

### PR DESCRIPTION
**Pull request by TarunAdobe**
## Description

Fixed track alignment in editable `sp-slider` with label-visibility="none"

## Related issue(s)

- #3812

## Motivation and context

slider line of sp-slider is not aligned in centre of the value input box when we dont have the label for slider (label-visibility="none" in sp-slider)

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_
    1. Go here
    2. Do this

## Screenshots (if appropriate)

Before - 
image

After - 
image

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: packages/slider/src/slider.css [hunk 1]
```diff
@@ -137,3 +137,8 @@ governing permissions and limitations under the License.
     padding: 0;
     margin: 0;
 }
+
+:host([label-visibility='none']) #track {
+    grid-area: number;
```
Comment (Westbrook) [inline packages/slider/src/slider.css:142]: Is there a path to allowing this to work while not changing the grid area?

Comment (TarunAdobe) [reply packages/slider/src/slider.css]: yes. `align-self: center` seems to do the same thing

**Comment by TarunAdobe** [comment]
> The GH Action failure is the only one here I don’t count as “flaky”. Can you see if it passes with #3792 before your changes?

There's only [one](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/18669/workflows/12ccd1e8-110b-491f-933e-f64215b9aef4/jobs/442363/tests) test failing right now which has "flaky" label. 
Also something is wrong with the VRT site I cannot access it. It shows site not found for me.

**Comment by Westbrook** [comment]
Not the sites for tests that fail are built. So check out https://4f1f073128af58796cfce3cb07338037--spectrum-web-components.netlify.app/review/ to see the annoying flakey failure here. 😰

**Review by Westbrook — COMMENTED** [review]
If you create a story for this, that will fail all the VRTs so you can see all of them. Being this is a client support request, it’s likely a good idea that we add it.

**Review by Westbrook — APPROVED** [review]
Excellent! :shipit:

**Pull request merged**